easybuild/easyconfigs/c/COSMO_pompa_crclim_dycore/COSMO_pompa_crclim_dycore-551f6b44dc13227d92225edfd6e3bfd58c782928-CrayGNU-18.08-cuda-9.1-double.eb

```diff
@@ -0,0 +1,38 @@
+# jg (cscs)
+easyblock = 'CMakeMake'
+
+name = 'COSMO_pompa_crclim_dycore'
+version = '551f6b44dc13227d92225edfd6e3bfd58c782928'
```

Comment:
We usually keep only the first 7 hash digits for the module version (to avoid such long file and module names)